crates/matrix-sdk/src/room/mod.rs

```diff
@@ -1432,7 +1436,7 @@ impl Room {
     pub fn send_raw<'a>(
         &'a self,
         event_type: &'a str,
-        content: serde_json::Value,
+        content: impl IntoRawMessageLikeEventContent,
```

Comment:
Hmm, why didn't we use `impl Into<Raw<...>>` here instead?

I just noticed, the example above has a `body` field for the `m.room.message` but is missing a `msgtype` field. Could you add that as part of the PR.

Reply:
Because coherence forbids `impl Into<Raw<SomeEvent>> for serde_json::Value` in a crate that is neither `serde_json` nor `ruma_common`.

Reply:
Ah right, thus we need a custom trait we have control over fully.